### ISSUE TITLE
🐛 Bug Report: Cannot find module 'pg-format'

### ISSUE THREAD

**Issue by awanlin** [backstage/backstage#26197]
### 📜 Description

While upgrading various plugins in the Community Plugin repo to `1.30.2` we have run into this error running tests:

```text
Cannot find module 'pg-format' from '../../../node_modules/@backstage/backend-common/dist/index.cjs.js'

    Require stack:
      /home/runner/work/community-plugins/community-plugins/workspaces/nomad/node_modules/@backstage/backend-common/dist/index.cjs.js
      service/router.test.ts

      17 | import express from 'express';
      18 | import request from 'supertest';
    > 19 |
         | ^
      20 | import { createRouter } from './router';
      21 | import { ConfigReader } from '@backstage/config';
      22 |

      at Resolver._throwModNotFoundError (../../../node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (../../../node_modules/@backstage/backend-common/dist/index.cjs.js:11:[14](https://github.com/backstage/community-plugins/actions/runs/10538939119/job/29201988099?pr=970#step:14:15))
      at Object.<anonymous> (service/router.test.ts:19:24)
```

PR with this failure are:

- https://github.com/backstage/community-plugins/pull/970
- https://github.com/backstage/community-plugins/pull/969
- https://github.com/backstage/community-plugins/pull/1012

### 👍 Expected behavior

The test should pass

### 👎 Actual Behavior with Screenshots

The test fail with this error: `Cannot find module 'pg-format' from '../../../node_modules/@backstage/backend-common/dist/index.cjs.js'`

### 👟 Reproduction steps

I'm not sure the exact steps here as out of the 60+ plugins we upgraded only 3 ran into this issue:

- https://github.com/backstage/community-plugins/pull/970
- https://github.com/backstage/community-plugins/pull/969
- https://github.com/backstage/community-plugins/pull/1012

### 📃 Provide the context for the Bug.

We'd like to be able to upgrade these plugins but this error is blocking them.

### 🖥️ Your Environment

Not sure in this case.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, but I'm happy to collaborate on a PR with someone else

**Comment by awanlin** [comment]
This came up on Discord but it was a different cases, they weren't running tests: https://discord.com/channels/687207715902193673/1276150117833113673

**Comment by freben** [comment]
Fixed by https://github.com/backstage/backstage/pull/26235

**Comment by maarten-vandeperre** [comment]
@freben ,
What should be changed exactly?

I have this now

`"dependencies": {
    "@backstage/backend-plugin-api": "^0.7.0",
    "@backstage/plugin-scaffolder-node": "^0.4.8",
    "@roadiehq/scaffolder-backend-argocd": "^1.1.28"
  },
  "devDependencies": {
    "@backstage/backend-common": "^0.23.3",
    "@backstage/cli": "^0.26.3",
    "@janus-idp/cli": "1.13.1"
  },`

**Comment by freben** [comment]
Upgrading Backstage as a whole is enough. But you could also make sure that your backend-common is resolved to the very latest version.

**Comment by benjdlambert** [comment]
Closing this as fixed, please upgrade to 1.30.4 using `yarn backstage-cli versions:bump`

**Comment by awanlin** [comment]
Just following up to confirm that using the latest patched version - `1.30.4` - does resolve the error noted in the plugins listed in the description. Thanks!
